extensions/resteasy-reactive/jaxrs-client-reactive/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/JaxrsClientReactiveProcessor.java

```diff
@@ -871,6 +910,212 @@ A more full example of generated client (with sub-resource) can is at the bottom
 
     }
 
+    private ResultHandle createMultipartForm(MethodCreator methodCreator, ResultHandle methodParam, Type formClassType,
```

Comment:
Can we get some comments on this? It's pretty large and I have no idea what it's supposed to do :)

Reply:
added `some` comments :)